fluss-lakehouse/fluss-lakehouse-paimon/src/main/java/com/alibaba/fluss/lakehouse/paimon/sink/NewTablesAddedPaimonListener.java

```diff
@@ -100,12 +103,8 @@ private Schema toPaimonSchema(TableInfo flussTable) {
         Schema.Builder schemaBuilder = Schema.newBuilder();
         Options options = new Options();
 
-        // only bucket exists, we set bucket num in paimon
-        // otherwise, it will be considered as dynamic bucket in paimon
```

Comment:
Shall we add bucket number and bucket key only when the table has bucket key @luoyuxia ? For example, should log table has a dynamic bucket in Paimon?

Reply:
@wuchong, @luoyuxia, https://github.com/apache/paimon/commit/c8f9b1860d37b7be25c97d5c842d5718d9cc9cd3 defines that `bucket-key` should be defined when the table has bucket. Therefore, it's better to use dynamic bucket for log append table.

Reply:
Yes, that's what I mean to do in comments, but miss in code..